pinecone/__init__.py

```diff
@@ -12,7 +15,12 @@
 
 # Kept for backwards-compatibility
 UpsertResult = None
+"""@private"""
 DeleteResult = None
+"""@private"""
 QueryResult = None
+"""@private"""
 FetchResult = None
+"""@private"""
 InfoResult = None
+"""@private"""
```

Comment:
These were showing up in the generated docs on the very top level page under the README. We could include them and/or comment them, but I hid them for now.

Reply:
Hiding seems good. There's nothing interesting to say about a bunch of `None` aliases.